tests/python_client/testcases/test_search.py

```diff
@@ -2034,6 +2037,65 @@ def test_search_expression_all_data_type(self, nb, nq, dim, auto_id, _async):
         assert (default_int64_field_name and default_float_field_name and default_bool_field_name) \
                in res[0][0].entity._row_data
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_search_with_comparative_expression(self, _async):
+        """
+        target: test search using comparison
+        method: test search using comparison
```

Comment:
This comment can not show what this case used for.

Reply:
> This comment can not show what this case used for.

The comment has been changed.